### PR TITLE
[Master] MCR for Grep and JDBC App 

### PR DESCRIPTION
**Pull request by AyeshaHamad**
Please review this MCR only. 
I have seen your review for the profile PR. But I have not updated it yet.

### CODE REVIEW

File: core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
+package ca.jrvs.apps.grep;
+
+//import com.sun.org.slf4j.internal.Logger;
+//import com.sun.org.slf4j.internal.LoggerFactory;
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
```
Comment (jarviscanada) [inline core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java:6]: remove

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      JDBCExecutor.logger.debug("Unable to delete record " + e);
+    }
+  }
+
+  private void commit(){
+    try {
+      this.connection.commit();
+      JDBCExecutor.logger.debug("CustomerDAO: Data commit is done successfully.");
+    }catch (SQLException e){
+      JDBCExecutor.logger.debug("CustomerDAO: Unable to commit data. " + e);
```
Comment (jarviscanada) [inline core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java:146]: JDBCExecutor.logger.debug("CustomerDAO: Unable to commit data. ", e);

**Review by jarviscanada — APPROVED** [review]
LGTM

File: core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java (hunk 1 above)
Comment (AyeshaHamad) [reply core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java]: done

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java (hunk 2 above)
Comment (AyeshaHamad) [reply core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java:146]: done

**Pull request merged**